lms/static/scripts/frontend_apps/services/grading.js

```diff
@@ -10,7 +10,7 @@ import { apiCall } from '../utils/api';
  * The fetched grade success result.
  *
  * @typedef FetchGradeResult
- * @prop {number} currentScore - The fetched grade
+ * @prop {number|null} currentScore - The fetched grade
```

Comment:
I added `null` because I confirmed that value empirically in Blackboard. Is it ever `undefined`?